CONTRIBUTING.md

```diff
@@ -87,6 +87,14 @@ pnpm pqb t select.test
 pnpm orm t belongsTo.test
 ```
 
+## VSCode Dev Container setup
+
+For an isolated dev environment that automatically starts postgres on launch for you:
+
```

Comment:
(I'm a total noob in VSCode)

On my linux, there are two versions of VSCode that pops out when searching for "vscode" in the package manager:
- OSS Code - package is officially maintained by Arch community and it is open-source.
- Visual Studio Code - package is maintained by some user, not as official, and it's not as open-source as the previous one. I guess, it's like Chromium and Chrome. Or MariaDB and MySQL.

And I found that there are no such `Dev Containers` extension in the OSS version.

So looks like this is available only in the proprietary VSCode.

Please add info that this requires a separate extension vscode:extension/ms-vscode-remote.remote-containers, and it's not available in OSS VSCode version.

Reply:
> (I'm a total noob in VSCode)
> 
> On my linux, there are two versions of VSCode that pops out when searching for "vscode" in the package manager:
> 
> * OSS Code - package is officially maintained by Arch community and it is open-source.
> * Visual Studio Code - package is maintained by some user, not as official, and it's not as open-source as the previous one. I guess, it's like Chromium and Chrome. Or MariaDB and MySQL.
> 
> And I found that there are no such `Dev Containers` extension in the OSS version.
> 
> So looks like this is available only in the proprietary VSCode.
> 
> Please add info that this requires a separate extension vscode:extension/ms-vscode-remote.remote-containers, and it's not available in OSS VSCode version.

Okay.  I highly recommend running the main VSCode for this reason.  It's still 100% free but Microsoft only holds on to some of the bits for their codespaces offering.  The actual remote containers code is actually completely open source so I'm not sure what's up there. 

That's the thing - you need to have it in your branch when you clone it.  So if you clone main into a dev container and then switch the branch to my PR, you can then do `F1` `Dev Containers: Rebuild Container`

Reply:
The irony here is that setting up dev containers after the fact is painful, but once you have it, it's the easiest way on Earth to immediately and safely start working on a project.

Reply:
Here's where to grab the official Microsoft version
https://code.visualstudio.com/download